rg_instructor_analytics/static/rg_instructor_analytics/js/EnrollmentTab.js

```diff
@@ -54,6 +54,7 @@ function EnrollmentTab(button, content) {
         };
 
         function onSuccess(response) {
+            console.log("hello",response);
```

Comment:
Please remove this log message or provide a more suitable description.

Reply:
done